rllib/core/learner/torch/torch_learner.py

```diff
@@ -208,9 +211,13 @@ def build(self) -> None:
                 self._module = TorchDDPRLModule(self._module)
             else:
                 for key in self._module.keys():
-                    self._module.add_module(
-                        key, TorchDDPRLModule(self._module[key]), override=True
-                    )
+                    if isinstance(self._module[key], torch.nn.Module):
```

Comment:
same here, use TorchRLModule

Reply:
or change instances of forward_train to `__call__`.

Reply:
done.